### DISCUSSION TITLE
How can I pause/play the partices?

### DISCUSSION THREAD

**Discussion by caseymockbee97** [matteobruni/tsparticles#4210] [Q&A] [answered]
I am looking for help on how to pause/play the particle animation like on [the example website](https://particles.js.org/). I'm using it in a react app and can't find anything about it in the docs. Thank you!

<img width="338" alt="image" src="https://user-images.githubusercontent.com/22242182/173115924-d8f7a993-2411-49d0-9d68-d2fb60756763.png">

Sorry if this is a question with an obvious answer!

**Comment by matteobruni** [comment, marked as answer]
If you want to pause the animation when clicking on the canvas you can see a sample configuration here: https://github.com/matteobruni/tsparticles/blob/main/demo/vanilla/public/presets/clickPause.json

I think there's a bug using buttons, but I need to debug it more. At least I've found a workaround using refs here: https://codesandbox.io/s/infallible-kate-332lc9?file=/src/App.tsx

**Reply by caseymockbee97** [reply]
Thank you! Unfortunately when I tried to implement clicking the canvas it would pause if anywhere on the screen was clicked. I ended up just setting motion and rotation animation to false and accepting the abrupt background shift from the state change. Maybe one day I'll get back to trying to debug this one.
